INFO     backend.layers.business.business:business.py:511 {'message': 'ingesting dataset', 'collection_version_id': 0ab69eb8-42d8-4597-b791-de1d0cb95791, 'url': 'http://fake.url', 'current_dataset_version_id': None}
INFO     backend.layers.business.business:business.py:511 {'message': 'ingesting dataset', 'collection_version_id': 0ab69eb8-42d8-4597-b791-de1d0cb95791, 'url': None, 'current_dataset_version_id': e3076e4d-8435-4f17-a708-aa8ccedde9dd}
INFO     processing:process_logic.py:45 Updating processing status
INFO     processing:process_logic.py:45 Updating processing status
INFO     processing:logger.py:26 Start create_artifact
INFO     processing:process_logic.py:45 Updating processing status
INFO     processing:process_logic.py:91 Uploaded [04486aa4-0402-4a7d-a6be-14805e038e6c/raw.h5ad] to s3://artifact_bucket/04486aa4-0402-4a7d-a6be-14805e038e6c/raw.h5ad
INFO     processing:process_logic.py:93 Updated database with raw_h5ad.
INFO     processing:process_logic.py:45 Updating processing status
INFO     processing:logger.py:28 Complete create_artifact
INFO     processing:process_logic.py:45 Updating processing status
INFO     processing:dataset_metadata_update.py:244 Main: Starting thread for h5ad update
ERROR    processing:dataset_metadata_update.py:281 Cannot find RDS artifact uri for e3076e4d-8435-4f17-a708-aa8ccedde9dd, and Conversion Status is not SKIPPED.
INFO     processing:process_logic.py:45 Updating processing status
INFO     processing:dataset_metadata_update.py:287 Main: Starting thread for cxg update
INFO     processing:process_logic.py:45 Updating processing status